iosapp/lib-ios/Sources/LibRecorder/SampleHandlerProxy.swift

```diff
@@ -4,11 +4,11 @@ import LibClients
 import os.log
 import ReplayKit
 
-// import UIKit
-
 public struct SampleHandlerProxy {
-  private var lastSavedDate = Date()
+  private var lastSavedDate = Date.now
   private let ciContext = CIContext()
+  private var previousScreenThumbnail: CGImage?
```

Comment:
this might very well be a good approach, but if the whole reasoning for moving this detection back into the recorder has to do with the app being quit, i had already thought of that and i think there's a pretty simple solution (and i have an issue #364 for it already). basically, we just respond to the app will terminate message and send a message to the filter to revoke any current suspensions. my guess is that will be sufficient.

still not back from my trip, but maybe we should try to chat soon, wednesday or thursday. 👍